tasks/util/proxy.py

```diff
@@ -176,4 +176,5 @@ def configure_docker_proxy(debug=False):
         input=json.dumps(user_config, indent=2).encode(), check=True)
 
     run("sudo systemctl daemon-reload", shell=True, check=True)
-    run("sudo systemctl restart docker", shell=True, check=True)
\ No newline at end of file
+    run("sudo systemctl restart docker", shell=True, check=True)
```

Comment:
@yoshiisato check this diff file. I am pretty sure you need to change the configuration of your text editor.